### PR TITLE
Update teams-powershell-release-notes.md

### PR DESCRIPTION
**Pull request by shriya999**

### CODE REVIEW

**Comment by shriya999** [comment]
@JohanFreelancer9 @get-itips @tiburd please review

File: Teams/teams-powershell-release-notes.md [hunk 1]
```diff
@@ -26,6 +26,7 @@ This page provides the latest Teams PowerShell change log for both General Avail
...
... expected to work similar to their remoting counter parts. The modern versions will be gradually rolled out. So some tenants will see the older remoting version (until the rollout completes).</li><li> Releases modernized versions of all 75+ Remove-Cs<Name>Policy cmdlets. These are expected to work similar to their remoting counter parts. The modern versions will be gradually rolled out. So some tenants will see the older remoting version (until the rollout completes).</li><li> Releases [Set\|Remove]-CsPhoneNumberAssignment cmdlets. These cmdlets will be gradually rolled out. So some tenants will get an error message referencing the existing cmdlets they can use (until the rollout completes).</li><li> Releases new parameter EnhancedEmergencyServiceDisclaimer for [Set\|New]-CsTeamsEmergencyCallingPolicy cmdlet.<li> Releases [Get\|Add\|Remove]-TeamChannelUser cmdlets.</li><li> Releases Export-CsOnlineAudioFile cmdlet.</li><li> Fixes error handlings for [Get\|Import\|Remove]-CsOnlineAudioFile cmdlets.</li><li>Fixes for Get-Team error handling. Outputs the error message if fetching teams data fails.</li><li>Fixes for Connect-MicrosoftTeams - AccessTokens expiry diff time increased.</li>
```
Comment (get-itips) [inline Teams/teams-powershell-release-notes.md:29]: @shriya999 please remove _en-us_ from the url paths in the included links, thank you

**Review by get-itips — APPROVED** [review]

**Review by rajgodwin — APPROVED** [review]
Approved

File: Teams/teams-powershell-release-notes.md [hunk 2]
```diff
@@ -26,6 +26,7 @@ This page provides the latest Teams PowerShell change log for both General Avail
...
```
Comment (tiburd) [inline Teams/teams-powershell-release-notes.md:29]:
```suggestion
| November 2021 | [3.0.0](https://www.powershellgallery.com/packages/MicrosoftTeams/3.0.0) |<li>Soon Microsoft Teams PowerShell Module versions in the 3.x.x series will be the only versions supported. All earlier versions will be on the deprecation path.</li><li> [BREAKING CHANGE] Get-CsOnlineUser and Get-CsOnlineVoiceUser: The following changes apply for TeamsOnly tenants:<ul><li>These cmdlets have now been migrated from the earlier implementation to the newer APIs.</li><li>(Using -Identity parameter): Attributes that are no longer relevant for Teams have been deprecated for TeamsOnly Tenants. Some attributes have also been renamed/replaced - Details [here](https://docs.microsoft.com/powershell/module/skype/get-csonlineuser?view=skype-ps) for Get-CsOnlineUser and [here](https://docs.microsoft.com/powershell/module/skype/get-csonlinevoiceuser?view=skype-ps) for Get-CsOnlineVoiceUser.</li><li> Get-CsOnlineUser (with -Filter parameter):  Formatting for filtering based on AssignedPlans and EnterpriseVoice has been modified. Filterable attributes have also been temporarily limited (based on usage) for TeamsOnly Users - Details [here](https://docs.microsoft.com/powershell/module/skype/get-csonlinevoiceuser?view=skype-ps).</li><li>These cmdlets will be gradually rolled out and some tenants will not experience these changes until the rollout is complete by the end of CY21.</ul></li><li>[BREAKING CHANGE] Get-CsTenant: Attributes that are no longer relevant in the Teams world have been deprecated for TeamsOnly tenants - Details [here](https://docs.microsoft.com/powershell/module/skype/get-cstenant?view=skype-ps).</li><li> Releases modernized versions of all 75+ Grant-Cs<Name>Policy cmdlets (all parameter sets). These are expected to work similar to their remoting counterparts. The modern versions will be gradually rolled out, so some tenants will see the earlier remoting version (until the rollout completes).</li><li> Releases modernized versions of all 75+ Remove-Cs<Name>Policy cmdlets. These are expected to work similar to their remoting counterparts. The modern versions will be gradually rolled out, so some tenants will see the earlier remoting version (until the rollout completes).</li><li> Releases [Set\|Remove]-CsPhoneNumberAssignment cmdlets. These cmdlets will be gradually rolled out, so some tenants will get an error message referencing the existing cmdlets that they can use (until the rollout completes).</li><li> Releases new parameter EnhancedEmergencyServiceDisclaimer for [Set\|New]-CsTeamsEmergencyCallingPolicy cmdlet.<li> Releases [Get\|Add\|Remove]-TeamChannelUser cmdlets.</li><li> Releases Export-CsOnlineAudioFile cmdlet.</li><li> Fixes error handlings for [Get\|Import\|Remove]-CsOnlineAudioFile cmdlets.</li><li>Fixes for Get-Team error handling. Outputs the error message if fetching teams data fails.</li><li>Fixes for Connect-MicrosoftTeams - AccessTokens expiry diff time increased.</li>
```

**Comment by tiburd** [comment]
@shriya999 @rajgodwin 
Hi! Can one or both of you please review my editorial changes to make sure that I didn't change the technical accuracy. The main change that needs to be checked is at the very end: "expiry diff time increased" > "expiration time is increased"
Thank you!

**Pull request merged**